frontend/src/components/Layout/Footer/index.tsx

```diff
@@ -0,0 +1,36 @@
+// import React, { FC } from "react";
+//
+// export const Footer: FC = () => {
+//   return <footer id={"BcFooter"}></footer>;
+// };
+// export default Footer;
+
+import React, { FC } from "react";
+import { Box, Grid, Link } from "@mui/material";
+import "./footer.scss";
+
+export const Footer: FC = () => {
```

Comment:
I'm pretty sure this wasn't in the scope of this ticket...

Reply:
You're right, but because we are using MUI some things need to be in place in order to work well later on.  We can discuss further but I will try to use placeholders more in the future to prevent out of scope tasks.